General-Application-Information.md

```diff
@@ -130,6 +130,78 @@ are interested in, it is a good idea to:
 used for searching the mailing list and linking to previous
 discussions.)
 
+## AI guidelines
+
+The use of Artificial Intelligence related tools ("AI" or "AIs" for
+short), like Large Language Models ("LLMs"), by developers is becoming
+more and more popular, but that's not a reason to blindly use them,
+especially when working on the Git project.
+
+### Legal issues
+
+In general, it's not clear if AI generated code or documentation are
+acceptable in the Git project. Authors contributing to Git are
+required to "sign off" the patches they contribute, and to agree with
+the Developer’s Certificate of Origin, also called DCO, (see
+https://git-scm.com/docs/SubmittingPatches#sign-off), while the DCO
+says basically that authors of a patch should be sure about the origin
+of its content.
+
+As we cannot usually be sure how AIs have been trained and that they
+are not just repeating proprietary existing code or docuementation
+they saw during their training, we cannot accept much AI generated
+code or documentation in general. Now, if it's only a few lines to fix
+a bug or to implement a common pattern or summarize something, and if
+that looks specific enough to a current concrete problem a developer
+is working on that might be OK.
```

Comment:
I would drop this, mostly because we clearly state that we're unsure of how the legality works out. Perhaps we can add something like:

> Only use such tools at your own discretion.

Reply:
I think it's a general rule about copyrighted work that if there are only a few ways, or a common way, to do something, then those cannot be copyrighted (see https://en.wikipedia.org/wiki/Sc%C3%A8nes_%C3%A0_faire).

I don't want to put that Wikipedia link in the doc, as I don't think it's worth distracting people with this.